.github/scripts/test-openjdk-pullrequests.py

```diff
@@ -801,7 +801,31 @@ def get_pr_to_test(untested_prs, visited):
         # Search runs for non-expired artifact
         for run in runs["workflow_runs"]:
             run_id = run["id"]
+            artifacts_obj_static = gh_api(["--paginate", f"/repos/{repo}/actions/runs/{run_id}/artifacts?name={_artifact_to_test}-static"])
+
+            for artifact in artifacts_obj_static["artifacts"]:
+                if not artifact["expired"]:
+                    # Download artifact
+                    artifact_id = artifact["id"]
+                    archive = Path(f"static_lib_{artifact_id}.zip")
+                    with open(archive, 'wb') as fp:
+                        gh_api([f"/repos/{repo}/actions/artifacts/{artifact_id}/zip"], stdout=fp)
+
+                    # Extract static-libs bundles
+                    try:
+                        with zipfile.ZipFile(archive, 'r') as zf:
+                            zf.extractall(path="extracted", filter="fully_trusted")
+                    finally:
+                        archive.unlink()
+
+                    break
+
+            if not (os.path.isdir("extracted") and os.listdir("extracted")):
+                untested_prs.setdefault("they are missing the static-libs bundle (added by JDK-8337265, JDK-8350443)", []).append(pr)
```

Comment:
I would make it a canary failure somehow if the JDK main bundle is available but the static-libs bundle is not. By now we can assume all PRs include JDK-8337265 right?

Reply:
as discussed offline, we still need to support the "JDK main bundle is available but the static-libs bundle is not" case